From 0a0123ec2e944d0e6b64ac6cebeaaffd7af9c3d2 Mon Sep 17 00:00:00 2001
Message-ID: <0a0123ec2e944d0e6b64ac6cebeaaffd7af9c3d2.1736059919.git.divya@subvertising.org>
In-Reply-To: <cover.1736059918.git.divya@subvertising.org>
References: <cover.1736059918.git.divya@subvertising.org>
From: Divya Ranjan <divya@subvertising.org>
Date: Sat, 4 Jan 2025 21:16:32 +0000
Subject: [PATCH 22/44] gnu: Add rust-nu-glob-0.88.

* gnu/packages/crates-io.scm (rust-nu-glob-0.88): New variable.
---
 gnu/packages/crates-io.scm | 20 ++++++++++++++++++++
 1 file changed, 20 insertions(+)

diff --git a/gnu/packages/crates-io.scm b/gnu/packages/crates-io.scm
index 19a18e04fb..d973843120 100644
--- a/gnu/packages/crates-io.scm
+++ b/gnu/packages/crates-io.scm
@@ -47310,6 +47310,26 @@ (define-public rust-nu-engine-0.88
     (description "This package provides Nushell's evaluation engine.")
     (license license:expat)))
 
+(define-public rust-nu-glob-0.88
+  (package
+    (name "rust-nu-glob")
+    (version "0.88.1")
+    (source
+     (origin
+       (method url-fetch)
+       (uri (crate-uri "nu-glob" version))
+       (file-name (string-append name "-" version ".tar.gz"))
+       (sha256
+        (base32 "1zb22s6d1x56h09jybwv2n5hfb9q7g39ngsy11s3a39yxqbf5ilg"))))
+    (build-system cargo-build-system)
+    (home-page "https://github.com/nushell/nushell/tree/main/crates/nu-glob")
+    (synopsis
+     "Fork of glob. Support for matching file paths against Unix shell style patterns.")
+    (description
+     "This package provides Fork of glob.  Support for matching file paths against Unix shell style
+patterns.")
+    (license (list license:expat license:asl2.0))))
+
 (define-public rust-nucleo-0.2
   (package
     (name "rust-nucleo")
-- 
2.47.1

